en/includes/references/conditional-auth/api-reference.md

```diff
@@ -1332,15 +1340,77 @@ It contains the necessary metadata for invoking the API when calling the callCho
   </tr>
 </table>
 
-If the consumer key and the consumer secret are added as secrets, they should be included in the ConnectionMetadata as aliases, as shown below.
+You can securely store consumer keys and secrets as **secrets** in conditional authentication scripts and refer to
+them in your conditional authentication scripts using the `secrets.key` syntax. For example, to retrieve a secret value, you may use:
+```angular2html
+var consumerSecret = secrets.clientSecret;
+```
+For more information on adding secrets, refer to the [Add a secret to the script]({{base_path}}/guides/authentication/conditional-auth/configure-conditional-auth/#add-a-secret-to-the-script) section in the 
+documentation.
+
+??? note "Change in behavior from 30th June 2024"
+    Starting from 30th June 2024, you are no longer required to set the aliases for consumer keys and consumer secrets when calling the callChoreo command. Instead, you may directly reference them using the `secrets.secretName` notation.
```

Comment:
This date should be changed right ?

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4773/commits/9acc22f1d66873e2c67cb71939862187674bece9

Reply:
There's one more entry

Reply:
addressed with https://github.com/wso2/docs-is/pull/4773/commits/cc28685065385ccc3454062eccfb6f0d6d5576f9